tdrs-backend/docker-compose.local.yml

```diff
@@ -67,6 +73,15 @@ services:
     depends_on:
       - localstack
       - postgres
+      - redis-server
+
+  redis-server:
+    image: "redis:alpine"
+    command: redis-server /tdpapp/redis.conf
+    ports:
+      - "6379:6379"
+    volumes:
+      - .:/tdpapp
```

Comment:
I've added these because `tdrs-start` was failing to connect backend to a non-existent redis instance so I've mirrored our `docker-compose.yml` additions to the local file as well.